backend/btrixcloud/orgs.py

```diff
@@ -679,14 +702,198 @@ async def get_org_slugs_by_ids(self):
             slug_id_map[org["_id"]] = org["slug"]
         return slug_id_map
 
+    async def export_org(
+        self, org: Organization, user_manager: UserManager
+    ) -> Dict[str, object]:
+        """Export all data related to org as JSON"""
+        org_out_export = OrgOutExport.from_dict(org.to_dict())
+        org_serialized = await org_out_export.serialize_for_export(user_manager)
+
+        cursor = self.crawl_configs_db.find({"oid": org.id})
```

Comment:
We also need to add pages here. Also a bit nervous about the size restriction, and what happens if there's more than 100,000 items... but I guess we need to start somewhere.

Perhaps for pages we allow more?

Reply:
Replacing these `cursor.to_list` calls with async iteration over the MotorCursor, which should remove this obstacle.